content/yaml-publishing/distribution.md

```diff
@@ -136,6 +136,31 @@ Note that Codemagic does not send status updates on the post-processing step. Yo
 
 Post-processing does not consume any build minutes.
 
+### Microsoft Partner Center
+
+{{<notebox>}}
+DISCLAIMER: Building Windows applications with Codemagic is in beta stages and will be free to all users with billing enabled until the end of 2021. Any feedback is well received in our [Slack channel](https://slack.codemagic.io).
+{{</notebox>}}
+
+Codemagic enables you to automatically publish your desktop app to the Microsoft Store. Note that a prior successful submission has to have been made through the Partner Center in order for Codemagic to publish your application.
```

Comment:
```suggestion
Codemagic enables you to automatically publish your Windows desktop app to the Microsoft Store. Note that a prior successful submission through the Partner Center is required for Codemagic to be able to publish your application.
```

Reply:
Add the same note here too that the msix package can be downloaded from build artifacts and uploaded manually